test/lan_test.rb

```diff
@@ -357,8 +357,8 @@ def expect_modification_succeedes(modname, method)
       end
 
       it "does not configure the interface if it is not connected" do
-        allow(Yast::Lan).to receive(:connected_and_bridgeable?).with(anything).and_return(false)
-        expect(Yast::Lan).not_to receive(:ProposeItem)
+        allow(Yast::Lan).to receive(:connected_and_bridgeable?).with(anything, anything, anything).and_return(false)
```

Comment:
```any_args``` doesn't fit?

Reply:
anything match single param, not list of params. In fact maybe better to remove whole "with".

Reply:
well I thought that having any_args is shorter than having anything 3 times. But removing with completely makes the same service in this case.